obspy/core/tests/test_response.py

```diff
@@ -538,12 +538,12 @@ def test_unknown_units_no_integration(self, testdata):
         assert resp.instrument_sensitivity.value == overall_sensitivity
         assert np.abs(data).tolist() == [overall_sensitivity] * len(freqs)
 
-    def test_unkown_units_PA_recalculate_sensitivity(self):
+    def test_unkown_units_PA_recalculate_sensitivity(self, testdata):
```

Comment:
Might as well fix this typo at the same time:
```suggestion
    def test_unknown_units_PA_recalculate_sensitivity(self, testdata):
```

Reply:
didnt see it thanks!